src/api/app/helpers/webui/webui_helper.rb

```diff
@@ -355,5 +355,40 @@ def pick_max_problems(remaining_checks, remaining_build_problems, max_shown)
     build_problems = remaining_build_problems.shift(show_builds)
     return checks, build_problems, remaining_checks, remaining_build_problems
   end
+
+  # responsive_ux:
+  def access_params
```

Comment:
I also find this name more than strange. Maybe `sign_up_url_params`?

Reply:
We chose `access_*` because it was used for both _log in_ and _sign up_ forms/views.

Now we created different helpers separating _log in_  and _sign up_  things. We created a new helper module called `AuthenticationProtocolHelper` for all these methods. It also includes code related to kerberos, proxy and non-proxy modes. So, we opted to merge your `ProxyModeHelper` into this new module.

 Note: feel free to propose better naming.

@hennevogel  :eyes: